### PR TITLE
Add capability for loading projects from paths with arbitrary project file names

### PR DESCRIPTION
**Pull request by olaldiko**
Add the capability to properly load projects from folders where the project file has a name different from "project.tscproj". Will search for first tscproj file in folder.
Ability to directly point to project files.

### CODE REVIEW

File: src/camtasia/project.py [hunk 1]
```diff
@@ -56,7 +56,12 @@ def timeline(self) -> Timeline:
     @property
     def _project_file(self):
         "The project's main JSON data file, i.e. the 'tscproj' file."
-        return self.file_path / 'project.tscproj'
+        if self.file_path.is_dir():
+            for file in self.file_path.iterdir():
+                if file.is_file() and file.suffix == '.tscproj':
+                    return file
+        else:
+            return self.file_path
```
Comment (abingham) [inline src/camtasia/project.py:65]: This needs to account for the possibility that a) file_path is a directory and b) no matching .tscproj file is found in it. As currently coded, this function will return `None`, which will lead to a confusing error message. It's probably sufficient to just raise a `FileNotFoundError`.

**Review by abingham — COMMENTED** [review]
This is mostly good, I just want one small change.

**Comment by olaldiko** [comment]
Hi @abingham , thank you very much for your feedback! I have just uploaded a new commit with the suggestion!

**Comment by abingham** [comment]
Thanks!

**Pull request merged**